ansible/roles/ps4-controller/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+--- 
+
+- name: Install git
+  become: true
+  apt: 
+    name: git 
+    state: latest
+
+- name: Put rules in /etc/udev/rules.d/
+  become: yes
+  command: wget https://raw.githubusercontent.com/chrippa/ds4drv/master/udev/50-ds4drv.rules -O /etc/udev/rules.d/50-ds4drv.rules
+
+- name: reload ds4drv rules
+  become: yes
+  command: udevadm control --reload-rules
+
+- name: activate rules
+  become: yes
+  command: udevadm trigger
+      
+- name: Install ds4drv (version with controller limit)
+  become: no
+  pip:
+    name: git+https://github.com/Benvii/ds4drv.git@193_limit_simultaneous_controllers
+    virtualenv: /home/pi/venv/
+    virtualenv_command: /usr/bin/python3 -m venv
+
+- name: reload systemctl
+  become: yes
+  command: systemctl daemon-reload
+
+- name: Copy systemd service file to server
+  become: yes
+  template:
+    src: ds4drv-service.j2
+    dest: /etc/systemd/system/ds4drv.service
+    owner: root
+    group: root
+  notify:
+    - Start ds4drv service
```

Comment:
You forgot the environment file `/etc/donkeycar/ds4drv.env` which contains the LED variable as explained in @Minemew2 [comment here](https://github.com/mdl29/donkeycarLPH/issues/33#issuecomment-1099174341)